app/models/provider_enrichment.rb

```diff
@@ -46,6 +46,17 @@ class ProviderEnrichment < ApplicationRecord
   validates :train_with_us, words_count: { maximum: 250 }
   validates :train_with_disability, words_count: { maximum: 250 }
 
+  validates :email, presence: true, on: :publish
```

Comment:
Could we put these into a one-liner or is there future benefit from splitting them out?

Reply:
One-liner should be fine 👍